cmd/kusk/cmd/install.go

```diff
@@ -35,7 +35,11 @@ import (
 	"strings"
 	"time"
 
+	"github.com/avast/retry-go/v3"
+	"github.com/ghodss/yaml"
```

Comment:
Why are you using this library instead of something Kubernetes (`sigs.k8s.io/yaml`)?

Also, surprised it's not appearing in the `go.mod`/`go.sum` changes in this PR.

Reply:
sigs.k8s.io/yaml doesn't work well so that's why we are using it cmd/kusk/cmd/generate.go, pkg/spec/spec.go and in this file.